js/index.js

```diff
@@ -1 +1,200 @@
 // Your code goes here
+
+function playSound(soundfile) {
+    document.getElementById("dummy").innerHTML= "<embed src=\""+soundfile+"\" hidden=\"true\" autostart=\"true\"loop=\"false\" />";
+  }
+
+
+const funBusHeading = document.querySelectorAll(".nav-link");
+const funBusImages = document.querySelectorAll("img");
+const destinationContainer = document.querySelectorAll("destination");
+const destinationButton = document.querySelectorAll(".btn");
+const theBody = document.querySelector("body");
+
+///This was my attempt at getting 
+///event propogation to work. 
+/// It just ended up breaking the page
+
+// body.addEventListener("click", e => {
+//     alert("What are you trying to click?");
+// })
+
+//// Home link
+
+funBusHeading[0].addEventListener("mouseover", e => {
+    funBusHeading[0].style.color = "red";
+    funBusHeading[0].style.background = "green";
+    funBusHeading[0].style.padding = "3%";
+    funBusHeading[0].style.borderRadius = "20%";
+    funBusHeading[0].style.boxShadow = "2px 17px 35px 1px #8E0000"
+    funBusHeading[1].style.fontSize = "10px";
+    funBusHeading[2].style.fontSize = "10px";
+    funBusHeading[3].style.fontSize = "10px";
+})
+
+funBusHeading[0].addEventListener("mouseout", e => {
+    funBusHeading[0].style.background = "white";
+    funBusHeading[0].style.color = "black";
+    funBusHeading[0].style.padding = "0%";
+    funBusHeading[0].style.boxShadow = "none";
+    funBusHeading[1].style.fontSize = "16px";
+    funBusHeading[2].style.fontSize = "16px";
+    funBusHeading[3].style.fontSize = "16px";
+})
+
+//// About Us link
```

Comment:
You can just use `.forEach` and apply to all of them when you `.querySelectorAll`